.github/workflows/main.yml

```diff
@@ -51,7 +88,7 @@ jobs:
         id: check_release
         run: |
           python -m pip install --upgrade pip
-          python -m pip install poetry githubrelease httpx==0.16.1 autopub
+          python -m pip install poetry githubrelease autopub
```

Comment:
Please revert the change to this line, since `httpx==0.16.1` is needed. (See relevant recent commit for its purpose.)